src/Core/src/Core/IIndicatorView.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.Maui.Graphics;
+
+namespace Microsoft.Maui
+{
+	public interface IIndicatorView : IView
+	{
+		int Count { get; }
+
+		int Position { get; set; }
+
+		double IndicatorSize { get; }
+
+		int MaximumVisible { get; }
+
+		bool HideSingle { get; }
+
+		Paint? IndicatorColor { get; }
```

Comment:
For now we only support solid colors, right? Could we add a comment?

Reply:
Sure.